### ISSUE TITLE
Error cocolani

### ISSUE THREAD

**Issue by cmdocmd** [ruffle-rs/ruffle#13388]
### Describe the bug

So after writing the username and press next when registering getting this error
![image](https://github.com/ruffle-rs/ruffle/assets/55096480/db42cf79-623a-4045-848d-d034b19c0f6f)

### Expected behavior

It should open another panel for continue registering

### Affected platform

Browser's extension

### Operating system

Pop os

**Comment by Lord-McSweeney** [comment]
Link to the game: https://safariislandsgame.com/play/?lang=en

I can't get to that step. After clicking "Join Now" and selecting a mask, after I click Next on the "SELECT YOUR TRIBE" page, I get this error:
```
Error occurred when running AVM2 frame script: TypeError: Error #1009: Cannot access a property or method of a null object reference. (accessing field: setText)
	at com.cocolani.panels::registration/preparepage()
	at com.cocolani.panels::registration/frame3()
	at flash.display::MovieClip/flash::display::MovieClip::gotoAndStop()
	at com.cocolani.common::simplebutton/enable()
	at com.cocolani.common::simplebutton()
	at Button_Rollover()
	at flash.display::MovieClip/flash::display::MovieClip::gotoAndStop()
	at com.cocolani.common::simplebutton/enable()
	at com.cocolani.common::simplebutton()
	at Button_Rollover()
	at flash.display::MovieClip/flash::display::MovieClip::gotoAndStop()
	at com.cocolani.common::simplebutton/enable()
	at com.cocolani.common::simplebutton()
	at Button_Rollover()
	at flash.display::DisplayObject/flash.display::DisplayObject()
	at flash.display::InteractiveObject/flash.display::InteractiveObject()
	at flash.display::DisplayObjectContainer/flash.display::DisplayObjectContainer()
	at flash.display::Sprite()
	at flash.display::MovieClip()
	at flash.display::MovieClip/flash::display::MovieClip::gotoAndStop()
	at com.cocolani.common::simplebutton/enable()
	at com.cocolani.common::simplebutton()
	at Bt_Randomize()
	at flash.display::MovieClip/flash::display::MovieClip::gotoAndStop()
	at com.cocolani.panels::registration/loadingInitListener()
	```

**Comment by Lord-McSweeney** [comment]
This has been fixed, the next issue is missing support for `URLLoaderDataFormat.VARIABLES`.

**Comment by cmdocmd** [comment]
fixed by #14044
